.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Description
```

Comment:
instead of `Description` use `Changes`. Also, add a comment to write these changes as bullet points.